packages/mask/src/extension/popups/pages/Personas/index.tsx

```diff
@@ -14,26 +16,42 @@ const PersonaSignRequest = lazy(() => import('./PersonaSignRequest/index.js'))
 const SelectPersona = lazy(() => import('./SelectPersona/index.js'))
 const AccountDetail = lazy(() => import('./AccountDetail/index.js'))
 const ConnectedWallets = lazy(() => import('./ConnectedWallets/index.js'))
+const ConnectWallet = lazy(() => import('./ConnectWallet/index.js'))
 
 const r = relativeRouteOf(PopupRoutes.Personas)
 const Persona = memo(() => {
+    const location = useLocation()
+    const modalNavigate = useModalNavigate()
+
+    useEffect(() => {
```

Comment:
It will cause a flash of an empty page if the router gets redirected while rendering.